patches/4D5388BF - Crackdown 2 Demo.patch.toml

```
@@ -0,0 +1,13 @@
+title_name = "Crackdown 2 Demo"
+title_id = "4D5388BF" # MS-35007
+hash = "54F6EF3FCB37D4C2" # default.xex
+#media_id = "22AFC3C2"
+
+[[patch]]
+    name = "Remove Demo Countdown"
```

If this freezes the timer it should have this name:
```suggestion
    name = "Freeze Demo Timer"
```